mozilla/fx-private-relay
# MPP-3328 & MPP-3327 - Premium Upsell

This PR fixes MPP-3328 and MPP-3327 


# New feature description
- MPP-3327 5 Mask Premium Upsell Header 
- Show relay metrics header to free users

# Screenshot (if applicable)

<img width="925" alt="image" src="https://github.com/mozilla/fx-private-relay/assets/3924990/06459016-50d6-4ca2-a735-18381c9bf990">


# How to test

MPP-3327

When a user hits their 5 mask limit take over the top part of the relay experience wit the following header. The header cannot be dismissed.

Do not show any other banners at the top of the experience (including if they don’t have the add-on)

CSAT banner can stay

Upgrade to premium link should be instrumented in GA (e.g., “upgrade-premium-header-mask-limit”)

Upgrade to premium CTA takes user to premium price plans 

Designs:

US:[How to upsell 5 mask limit](https://www.figma.com/file/i0SNijb8Twcn7czbzcDLYj/How-to-upsell-5-mask-limit?type=design&node-id=337-7972&mode=design&t=ABRogTnrkrIIUdMH-4) 

Non-US: [How to upsell 5 mask limit](https://www.figma.com/file/i0SNijb8Twcn7czbzcDLYj/How-to-upsell-5-mask-limit?type=design&node-id=513-9586&mode=design&t=qwstSpGIsDdEs5vE-4)

MPP-3328
Show free users a similar welcome and metrics like premium users see that include (email masks used, emails blocked, emails forwarded, and trackers removed)

US/Canada version: [How to upsell 5 mask limit](https://www.figma.com/file/i0SNijb8Twcn7czbzcDLYj/How-to-upsell-5-mask-limit?type=design&node-id=337-7972&mode=design&t=iXRryaVFKXuvyjcT-4)  

Global version: [How to upsell 5 mask limit](https://www.figma.com/file/i0SNijb8Twcn7czbzcDLYj/How-to-upsell-5-mask-limit?type=design&node-id=337-8443&mode=design&t=iXRryaVFKXuvyjcT-4)

# Checklist (Definition of Done)
